x-pack/platform/plugins/shared/maps/public/classes/styles/color_palettes.ts

```diff
@@ -49,6 +49,7 @@ function getColorBlindPalette(steps: number) {
 
 const COLOR_PALETTES: COLOR_PALETTE[] = [
   {
+    title: 'Blues',
     value: 'Blues',
     getPalette: (steps: number) => {
       return euiPaletteCool(steps);
```

Comment:
As for this case in particular I don't know why they are using the `COLOR_PALETTE` type. Seemingly all the use cases are just using directly and never passing to the `EuiColorPalettePicker`. I think this should be fine as you have it or could just set it to `''` to match the old behavior.